### ISSUE TITLE
misunderstandable - unclear "install mod instead"

### ISSUE THREAD

**Issue by Dreifels** [Nexus-Mods/Nexus-Mod-Manager#1138]
Old Problem, but now I want to ask to fit it by a better description:
Problem:
I install a mod with BSAs or loose, anyways, but esp or esm And addtional files
Later I want to update the esp(esm) only, let all other valid.
= a simply replace of the old esp by the new one, nothing more.

The message shown is confusing:
ask if I want to install the "mod" "instead", and when I say yes, all additional files no longer are connected well.
There has to be a clear question if the esp/esm is to replace ONLY

"install mod instead" and "normal" are to unclear phrases (althought I do use NMM for years, I still have to think twice what to choose and verify after, so what nerfs, no tuser-friendly and for non-english users hard to understand well.)

Edit:
and just now I run into the trap again:
Installed "BecomeJarlOfIvarstead_1_3 SSE" and after that the update, that is the esp only "BecomeJarlOfIvarstead_1_3.1 SSE DV 1.5 esp_only" and when I checked it, active is the esp of the 1.3 and the esp of 1.3.1 is not there in /Data/

Edit 2:
and more bad: I click on the 1.3.1 and say reinstall, and nothing happens.

**Comment by DuskDweller** [comment]
Sorry, I don't get it, can you rephrase it ?

**Comment by Dreifels** [comment]
now you see me without understanding.
Ok, try to explain with other word, so far possible.

you have installes a mod with an esp and BSAs or loose files.
you get an update for the esp (only) (esp has same name)
when you click to install you get window asking if you want to "activate instead" 
If you do so, the BSAs or loose files no longer get loaded, buzt the new esp only.
If you say no, then the new esp get does not replace the old esp but is simply not installed.
If you choose "update" the new esp gets installed but no longer the loose files or BSAs get loaded. The "updated" deletes ALL of the former mod. (not phsically, but it gets no longer loaded.)
there must be a clear option to "replace" the esp ONLY but keeep all other active loaded, just now as part of the new esp.

**Comment by DuskDweller** [comment]
Thanks, now I got it!

**Comment by DuskDweller** [comment]
Are you installing the German or the English version ?

**Comment by Dreifels** [comment]
English, from here, never seen upgrade option to German version. and here also is none, or?

**Comment by DuskDweller** [comment]
The English Mod is not standalone, you need to install the German version first.

**Comment by Dreifels** [comment]
sorry? 1) I do not know where to download a German version. What do you mean with "sand alone"?
2) for years, I simply click on update when I get a message for it and let it update, and that is English.
![000](https://user-images.githubusercontent.com/40837496/115859916-0d64e180-a431-11eb-9ebd-6e5768693791.jpg)

**Comment by DuskDweller** [comment]
That mod doesn't work on its own, you need to download and install this first:
- https://www.nexusmods.com/skyrimspecialedition/mods/48645

and then this one you got is the English translation of that mod:
- https://www.nexusmods.com/skyrimspecialedition/mods/48650?tab=files

**Comment by Dreifels** [comment]
this I have for years and it works fine
bzt see screenshot I made just now, that is the prlobelm I complain: the wording / deacription is confsusing
![hc_003](https://user-images.githubusercontent.com/40837496/115860761-42bdff00-a432-11eb-9d87-e396c7e1e994.jpg)

**Comment by Dreifels** [comment]
Sir,  I am the Dreifels, that one who made the conversion and the translation(s). No need to say me what to do to get the mod working.
I complain **your** words in the NMM window above, not the mod.
(ovrall, it is same like simple replace the esp of the former version by the new esp, that's all. But doing htis manually does not store it in NMM. There is a way in NMM, but in nebula of **your** wordings.

**Comment by DuskDweller** [comment]
Oh sorry, I didn't know your were the Dreifels.
Please help me shed some light on the nebula of my words, which wording will you word for those words ?

**Comment by Dreifels** [comment]
**Converntion to the choice of words and understanding of the terms**

1) a "**Mod**" may be one or more esp (or esm) along with addtionial .BSAs and/or loose files which has to get installed together with its own dependencies.

by that a question for "install **_instead_**" is confusing and wrong meaning in any case. A "**Mod**" can get only uninstalled and deleted (complete)

2) an "**Update**" means, that **_parts_** of a "**Mod**" (as definition in (1)) get **replaced** by newer files. This can be a new esp or esm only, this also can be a new BSA or loose files, but in any case that are replacement(s) of files exising and installed already and have same name.

In fact, that only is an **overweriging** by files with same name but newer date, replacing files with same name and earlier date. That's all. There is **NO new installation of a mod**, no new dependencies, all in the installation has to keep unchanged, except that these files now are of newer date. 

3) By that, an installation of one mod "instead" by another mod must not be and is irregular. (remember, what a "Mod" is, (1).

So you have to give these alternative Options ONLY:

1) **INSTALL** a "**Mod**" NEW (and then check if there are conflicts with exisiting files etc.)

2) **UPGRADE** a "Mod" => REPLACE (or add) files (of an existing installation) of same names by newer files of same names, or add not existing files dependent to an existing esp/esm (in this case the .zip always contains also the esp/esp with same name of an exxisting installed esp/esm) 

**that is NOT a new installation and NOT an installation of a Mod INSTEAD.**

By that **UPGRADE** (2) NMM has to look for file names being in the .zip of the upgrade to install and search if same file names exist already. Or, if there are files in the .zip not existing already to what esp/esm they dependent. 

3) **DELETE** a Mod - this means, ALL files, incl. later upgrades, get deleted complete, so as the mod never had been installed. (check also Cache and Virtual)

Samples for different installations and upgrades

a new Mod's .zip contains (i.e.)

BecomeJarlOfIvarstead_1_3 SSE DEUTSCH komplett.7z contains
LC_BecomeJarlofIvarstead - Textures.bsa
LC_BecomeJarlofIvarstead.bsa
LC_BecomeJarlofIvarstead.esp

this (**all together**) is the "**Mod**"

It gets an **UPGRADE** (or Update, both words are fine in this case)
BecomeJarlOfIvarstead_1_3.1 SSE DV 1.6 esp_only.7z contains
LC_BecomeJarlofIvarstead.esp

This is NOT a "Mod", but an "Upgrade" file for an existing Mod, already installed.

This is NOT an "Install instead" of a Mod, but a "Replacement" (overwrite) of one (or more) FILE(s) (not of a MOD !) to an existing Mod!

So the question "do you want to install this Mod INSTEAD" ist wrong and total confusing.
Question must be if user wants to REPLACE the file(s) of the installed Mod by the newer file(s) of the Upgrade, but ONLY the files, not the Mod!!!, = overwrite, nothing else.

That message window of NMM (screenshot above) it totally confusing and under does not understand well that to do! (a main reason for users not to use NMM but Vortex, as the want not confusing messages)

OK now?
